apps/schedules/lib/repo.ex

```diff
@@ -20,18 +20,23 @@ defmodule Schedules.Repo do
   @spec by_route_ids([Route.id_t()], Keyword.t()) :: [Schedule.t()] | {:error, any}
   def by_route_ids(route_ids, opts \\ []) when is_list(route_ids) do
     opts = Keyword.put_new(opts, :date, Util.service_date())
+    no_cache = Keyword.get(opts, :no_cache)
 
     @default_params
     |> Keyword.put(:route, Enum.join(route_ids, ","))
     |> Keyword.put(:date, Keyword.fetch!(opts, :date) |> to_string())
     |> add_optional_param(opts, :direction_id)
     |> add_optional_param(opts, :stop_sequences, :stop_sequence)
     |> add_optional_param(opts, :stop_ids, :stop)
-    |> cache(&all_from_params/1)
+    |> cache_condition(no_cache)
     |> filter_by_min_time(Keyword.get(opts, :min_time))
     |> load_from_other_repos
   end
 
+  # Will almost always cache, unless the calling function explicitly passes "no_cache"
+  defp cache_condition(params, nil), do: cache(params, &all_from_params/1)
+  defp cache_condition(params, _no_cache), do: all_from_params(params)
+
```

Comment:
🤪 the pessimist in me is thinking in the worse cast scenario, someone somewhere might pass in a `no_cache: false` here, which would produce the undesired result.
Might want to tweak these function headers a bit, OR adjust line 23 to define no_cache as a boolean, e.g. something like

```elixir
no_cache = Keyword.get(opts, :no_cache) == true
```